File: programme/area-of-triangle/AreaOfTriangle.java
// Write a programme to find area of triangle
package com.company;

import java.util.Scanner;

public class AreaOfTriangle {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a : ");
        int a = sc.nextInt();

        System.out.print("Enter b : ");
        int b = sc.nextInt();

        System.out.print("Enter c : ");
        int c = sc.nextInt();

        area(a, b, c);

        sc.close();

    }

    public static int area(int a, int b, int c) {

        int s = (a + b + c) / 2;

        int area = (int) Math.sqrt(s * (s - a) * (s - b) * (s - c));
        System.out.println("\nArea of triangle : " + area);
        return area;

    }

}

File: programme/area-of-triangle/area-of-triangle.js
// Write a programme to find area of triangle

"use strict";

function area(a, b, c) {
  let s = (a + b + c) / 2;
  let product = Math.sqrt(s * (s - a) * (s - b) * (s - c));

  return product;
}

let a = prompt("a : ");
let b = prompt("b : ");
let c = prompt("c : ");
let s = (+a + +b + +c) / 2;

console.log("\ns :", s);

console.log("\narea :", area(+a, +b, +c));
